### PR TITLE
Adding different components types

### PR DESCRIPTION
**Pull request by nadiam84**

### CODE REVIEW

File: guides/react/app/apps/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Apps
+
+Maps directly to the [organisms](http://bradfrost.com/blog/post/atomic-web-design/#organisms)
+concept of atomic design
+
+Any app is a composition of components
```
Comment (treshugart) [inline guides/react/app/apps/README.md]: Maybe:

> An app is a component composed of other components

or something like that?

File: guides/react/app/components architecture/components-types/services/README.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+```
+
+## Basic service: data manipulation
+
+Same situation as basic fetch services, is for when there is a need to perform some data
+manipulations (like perform a POST/PUT request): all this logic could be encapsulated within a
+DataProvider, which again, accepts props and returns result of the manipulations and API to perform
+those manipulations via render props.
+
+```
```
Comment (treshugart) [inline guides/react/app/components architecture/components-types/services/README.md]: would be good to add language annotations to these:

~~~
```js
~~~

File: guides/react/app/apps/README.md (hunk 1 above)
Comment (nadiam84) [reply guides/react/app/apps/README.md]: Yep, looks better, will fix in a follow-up PR

File: guides/react/app/components architecture/components-types/services/README.md (hunk 2 above)
Comment (nadiam84) [reply guides/react/app/components architecture/components-types/services/README.md]: Will fix in a follow-up PR 👍

**Pull request merged**